{{ cookiecutter.formal_name }}/briefcase.toml

```diff
@@ -3,6 +3,6 @@
 app_path = "src/app"
 app_packages_path = "src/app_packages"
 support_path = "src"
-{{ {"3.8": "support_revision = 10", "3.9": "support_revision = 13", "3.10": "support_revision = 7"}.get(cookiecutter.python_version|py_tag, "") }}
+{{ {"3.8": "support_revision = 10", "3.9": "support_revision = 13", "3.10": "support_revision = 8", "3.11": "support_revision = 0"}.get(cookiecutter.python_version|py_tag, "") }}
```

Comment:
The multiline formatting is definitely worthwhile; I've applied that change. However, it's not possible to factor out the `support_revision` portion (at least, not without some additional logic, either in the template, or in Briefcase). 

Briefcase uses the existence of the `support_revision` key to determine if the version is supported at all; if the template needs a support package, and there's no `support_revision` listed, you get a clean error message saying the version isn't supported. 
```
[localtest] Installing support package...

Unable to download macOS support package for Python 3.12 on arm64.

This is likely because either Python 3.12 and/or arm64
is not yet supported on macOS. You will need to:
    * Use an older version of Python; or
    * Compile your own custom support package.
```

If you omit just the version, `support_revision = ` is a syntax error; If you use an empty string (`support_revision = ""`), you get an attempt to download a URL that doesn't exist:
```
[localtest] Installing support package...
Using support package https://briefcase-support.s3.amazonaws.com/python/3.12/macOS/Python-3.12-macOS-support.b.tar.gz

Unable to download macOS support package for Python 3.12 on arm64.
...
```

(note the `b.tar.gz` URL). 

For my money, it's better to not report the bad URL to the user; and I'm not wild about `support_revision = ""` as a marker of "not supported" status. Adding `if python_version >= 3.8 and < python_version < 3.12` logic also seems annoyingly complicated given the circumstances. Yes, there's a little repetition in the template; however, with the reformatting of the dictionary block, I don't think it's *that* bad, given the alternatives.